regen/opcode.pl

```diff
@@ -1045,7 +1045,7 @@ END
 END
 
 for (@ops) {
-    print $oc "\t", tab(3, "Perl_$check{$_},"), "\t/* $_ */\n";
+    print $oc "    ", tab(3, "Perl_$check{$_},"), "    /* $_ */\n";
```

Comment:
That `tab()` function comes from `regen/regen_lib.pl`, and should presumably also change; the uses of the nearby `wrap()` function look like they may also introduce tabs.